service/src/main/kotlin/fi/espoo/evaka/assistanceneed/vouchercoefficient/AssistanceNeedVoucherCoefficientController.kt

```diff
@@ -48,19 +49,27 @@ class AssistanceNeedVoucherCoefficientController(
                         Action.Child.CREATE_ASSISTANCE_NEED_VOUCHER_COEFFICIENT,
                         childId,
                     )
-                    adjustExistingCoefficients(tx, childId, body.validityPeriod, null)
-                    tx.insertAssistanceNeedVoucherCoefficient(childId, body).also {
-                        asyncJobRunner.plan(
-                            tx,
-                            listOf(
-                                AsyncJob.GenerateFinanceDecisions.forChild(
-                                    childId,
-                                    body.validityPeriod.asDateRange(),
-                                )
-                            ),
-                            runAt = clock.now(),
-                        )
-                    }
+                    adjustExistingCoefficients(
+                        user,
+                        clock.now(),
+                        tx,
```

Comment:
optional minor style improvements: 

1. tx has usually been the first parameter
2. using named variables is preferred especially when the value such as null or true/false does not tell what it is. Intellij at least has a quick fix available to add name to all function argumets